src/command/defaults/WhitelistCommand.php

```diff
@@ -116,7 +123,11 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
 					return true;
 				case "remove":
 					if($this->testPermission($sender, DefaultPermissionNames::COMMAND_WHITELIST_REMOVE)){
-						$sender->getServer()->removeWhitelist($args[1]);
+						$server = $sender->getServer();
+						$server->removeWhitelist($args[1]);
+						if($server->hasWhitelist()){
```

Comment:
`!isWhitelisted()` would be more appropriate here, in case some plugin alters the whitelist behaviour. We don't currently support this, but might in the future.

Reply:
Done